File: age_verifier.rb
def verify(age)
  if age.class == Array
    age.each do |num|
      if num < 21
        puts "Too Young"
      else
        puts "Proceed"
      end
    end
  else
    if age < 21
      puts "Too Young"
    else
      puts "Proceed"
    end
  end
end

verify([19, 16, 9])
verify(25)

File: ageVerifier.js
function verify(age) {
  if (Array.isArray(age)) {
    for(var i = 0; i < age.length; i++) {
      if (age[i] < 21) {
        console.log("Too Young");
      } else {
        console.log("Proceed");
      } // cond
    } // for loop
  } else {
    if (age < 21) {
      console.log("Too Young");
    } else {
      console.log("Proceed");
    } //cond
  } // else
}; // func

verify([1, 29, 9])
verify(39)
